engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/details/content-host-details-info.controller.js

```diff
@@ -129,5 +129,9 @@ angular.module('Bastion.content-hosts').controller('ContentHostDetailsInfoContro
 
             return ids.join(" or ");
         };
+	
+        $scope.convertMemToGB = function (memoryValue) {
```

Comment:
@snagoor since you don't have a dev env, i wrote a test for you here:  https://paste.fedoraproject.org/525127/70510148/raw/

it also fixes a couple of styling issues (mainly extra white space)